SnapshotTesting.xcodeproj/project.pbxproj

```diff
@@ -647,7 +671,11 @@
 			buildSettings = {
 				BUNDLE_LOADER = "$(TEST_HOST)";
 				INFOPLIST_FILE = Tests/Info.plist;
-				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
+				LD_RUNPATH_SEARCH_PATHS = (
```

Comment:
We auto-create the Xcode project file using XcodeGen. Can you run `make xcodegen` from the project directory and recommit that file?

Reply:
Done!